bin/run.sh

```diff
@@ -0,0 +1,3 @@
+npm install
```

Comment:
This doesn't automatically happen via the Node buildpack?

https://github.com/cloudfoundry/nodejs-buildpack/blob/5a0277f8a3a260f1a989af16c3208a0ef5459fc7/lib/dependencies.sh#L13

Also, `npm install` is already being called in `update_cg_style`.